.github/workflows/main.yaml

```diff
@@ -65,16 +68,44 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
+      - name: Run Networked Tests against anvil
+        # run separately to clarify its dependency on outside binary
+        run: pytest -n auto tests/integration/network/anvil/
+
+  integration:
+    name: "integration tests (fork mode and Sepolia)"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' || github.event_name == 'pull_request_target'
+    steps:
+      - name: Check if the user is a contributor
```

Comment:
i think we can actually grab the merge commit so that `pull_request_target` behaves better
https://github.com/actions/checkout/issues/518